Networking/Networking/Logging.swift

```diff
@@ -24,92 +24,107 @@ internal func resetDefaultDebugLevel() {
     defaultDebugLevel = CocoaLumberjack.dynamicLogLevel
 }
 
-internal func DDLogDebug(_ message: @autoclosure () -> String,
-                         level: DDLogLevel = defaultDebugLevel,
-                         context: Int = 0, file: StaticString = #file,
-                         function: StaticString = #function,
-                         line: UInt = #line, tag: Any? = nil,
-                         asynchronous async: Bool = true,
-                         ddlog: DDLog = DDLog.sharedInstance) {
-    CocoaLumberjack.DDLogDebug(message(),
-                               level: level,
-                               context: context,
-                               file: file,
-                               function: function,
-                               line: line,
-                               tag: tag,
-                               asynchronous: async,
-                               ddlog: ddlog)
```

Comment:
I can't tell which rule is overwriting this formatting, but the added suggested code (below) is much less skimmable and less readable.